docs/README.md

```diff
@@ -1,9 +1,14 @@
-<div align="center">
-  <img alt="OpenSauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
-  <h1>🍕 Open Source Education Path with OpenSauced 🍕</h1>
+
+<div style="text-align: center; padding: 20px;">
+    <h1 style="font-size: 3em; font-family: 'Arial', sans-serif; font-weight: bold; margin: 10px 0;">
+       The Open Source Education Path from
+ <span style="background: linear-gradient(to right, #e74c3c, #e67e22); -webkit-background-clip: text; color: transparent;">
+            OpenSauced
+        </span>    </h1>
 </div>
 
-Welcome to the Open Source Education Path with OpenSauced!
+
+**Welcome to the Open Source Education Path with OpenSauced!**
```

Comment:
Do we want "with" or "from" OpenSauced here?

Because the main title is Open Source Education Path "from" OpenSauced.

Reply:
For consistency, I think you should stick with "from" @adiati98. What do you think @BekahHW?